test/test_latex_to_pdf.rb

```diff
@@ -43,11 +43,15 @@ def test_broken_doc_halt_on_error
     end
   end
 
-  def test_broken_doc_overriding_halt_on_error
```

Comment:
That is a good topic to discuss. Sometimes, it is more user-friendly to provide at least some pages if the source file is somewhat broken.

Anyways, there should be a better and nicer mechanism to inform about latex errors. Maybe some sort of a tex output file parsing?